src/App.js

```diff
@@ -1,12 +1,38 @@
+import React from 'react';
+import {
+  BrowserRouter as Router, Routes, Route, Link,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
 import QuoteDisplay from './components/Quotes';
+import Home from './components/Home';
+import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <Calculator />
-      <QuoteDisplay />
-    </div>
+    <Router>
+      <div className="App">
+        <nav>
+          <ul>
+            <li>
+              <Link to="/">Home</Link>
+            </li>
+            <li>
+              <Link to="/calculator">Calculator</Link>
+            </li>
+            <li>
+              <Link to="/quote">Quote</Link>
+            </li>
+          </ul>
+        </nav>
+
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/calculator" element={<Calculator />} />
```

Comment:
- [ ] There should be a text aligned to the left of the calculator as seen the the given wireframe. The calculator shouldn''t be centered. 
- [ ] The calculator should be aligned to the right and the text besides it aligned to the right of the page.

|   |   |
| ------------- | ------------- |
| Your Design | <img width="916" alt="image" src="https://github.com/roodjinocherilus/magicmath/assets/52934128/c50eb0a9-c8fc-4991-8af4-6f0c1e28313d">  |
| Given Wireframe  | ![image](https://github.com/roodjinocherilus/magicmath/assets/52934128/7b26ebcc-98fb-4164-a4a1-405f20abb3c1)
  |